requirements/prod.txt

```diff
@@ -0,0 +1,46 @@
+argparse==1.4.0
```

Comment:
Are we good with these versions @Eugene-hu  @opentaco  @isabella618033 @unconst  ??

I would like to review them well so we agree this versions are expected versions for bittensor

Reply:
I checked some existing bittensor installations and most of their packages have newer versions than what is pinned here. This just means we'll have to do some more work in future to advance the pinned version safely to the latest, which may offer improved package benefits, while we ensure bittensor doesn't break.

Reply:
Great great.
I don't especially want these versions, we can change them to the ones that best fit us